packages/modul-components/src/components/radio-group/radio-group.ts

```diff
@@ -72,35 +90,75 @@ export class MRadioGroup extends BaseRadioGroup implements RadioGroup {
         this.model = value;
     }
 
-    protected created(): void {
-        this.internalValue = undefined;
+    public onFocus(event: FocusEvent): void {
+        if (!this.internalIsBlur) {
+            this.emitFocus(event);
+        }
+        this.internalIsFocus = true;
+    }
+
+    public onBlur(event: FocusEvent): void {
+        this.internalIsFocus = false;
+        this.internalIsBlur = true;
+        setTimeout(() => {
+            if (!this.internalIsFocus) {
+                this.emitBlur(event);
+            }
+            this.internalIsBlur = false;
+        });
     }
 
     @Watch('value')
     private onValueChange(value: any): void {
         this.internalValue = value;
     }
 
+    @Watch('focus')
+    private focusChanged(focus: boolean): void {
+        const selector: string = this.as<InputStateInputSelector>().selector || 'input, textarea, [contenteditable=true]';
+        const elements: NodeListOf<Element> = this.$el.querySelectorAll(selector);
+        let inputEl: HTMLElement | undefined;
+
+        // Find right element to focus
+        if (elements.length > 1) {
+            elements.forEach((radio) => {
+                if ((radio as unknown as MRadio).value === this.value) {
```

Comment:
le selecteur retourne des HTMLInputElement et non des MRadio, l'assertion n'est pas bonne ici. J'aurais plutôt fait

```
            elements.forEach((radio: Element) => {
                if ((radio as HTMLInputElement).value === this.value) {
                    inputEl = radio as HTMLElement;
                }
            }); 
```

Si c'est vraiment des MRadio que tu veux, au lieu d'utilisé un sélecteur tu peux faire directement

`this.$children ` tel que décrit dans l'api https://vuejs.org/v2/api/#vm-children je pense que ca serait mieux dans ce cas.